package.json

```diff
@@ -37,6 +40,8 @@
     "expr-eval": "^2.0.2",
     "gsap": "^3.11.3",
     "harp": "^0.46.0",
+    "jest": "^29.3.1",
```

Comment:
Do we need Jest? vue test utils uses it internally but we don't need to add it to our project

Reply:
this should go to the dev-dependencies. We're setting the version here because the vue-cli-service version is not ready for 2.7 yet.

Reply:
Tested with @vue/vue2-jest. It gets installed.